ibis/backends/tests/test_interactive.py

```diff
@@ -84,12 +84,11 @@ def test_interactive_non_compilable_repr_does_not_fail(table):
     repr(table.string_col.topk(3))
 
 
-def test_histogram_repr_no_query_execute(table, queries):
-    tier = table.double_col.histogram(10).name("bucket")
-    expr = table.group_by(tier).size()
-    expr._repr()
-
-    assert not queries
+# def test_histogram_repr_no_query_execute(table, queries):
+#     tier = table.double_col.histogram(10).name("bucket")
+#     expr = table.group_by(tier).size()
+#     repr(expr)
+#     assert not queries
```

Comment:
This test case doesn't make much sense since the `queries` fixture sets the interactive repr on. 

I disabled the test case because the expression fails to execute, so we need a follow-up for this.

Reply:
Well, the point was to have a test for the code path that never executes, but you've combined that all into `__repr__`.

If this was the only problematic test I'd say just delete it.

Reply:
Removed.